src/main/clojure/aerospike_clj/policy.clj

```diff
@@ -66,23 +66,24 @@
 
 (defn ^WritePolicy write-policy
```

Comment:
In general, I see many functions that are internal to their namespace and are declared as public.
It would be a good idea to run through the project and make every function that doesn't need to be public private.

Reply:
Done.